### DISCUSSION TITLE
How do I create defaults for map or list types?

### DISCUSSION THREAD

**Discussion by sid137** [tywalch/electrodb#273] [Q&A]
I'm trying to create a default value for a map attribute, and its not working.  

[Example](https://electrodb.fun/?#code/PQKgBAsg9gJgpgGzARwK5wE4Es4GcA0YuccYGeqCALgUQBYCG5YA7llXWAGbZwB2MXGBDAAUKKwBbAA5QMVMAG8wAUT5V2AT0IBlTADcsAY1IBfbhiiSwAIkRwjVSzABGNgNzijUPrgVUGFwRSAF5bTShUDAB9AKC4aL4GSTgPcVAwABUGXABrITUNKk1hMW9ffxz8sDC+OBZVdS0AClEwJTb2sElYRAAuDq6u-iLNAZsAvNwbfE6h-UxcLB9xgEYZua7iDEMTccnchmlpG03TWaGGKicsF1QqPAHFTfaH5KeXruLpOHG-bD4AHMNkMhuQ0FhyDABk50J9zp8Dh9QV9ND8-jcgSCUWQ4BCoTCMHCUQiUdJLAArBxUZE476-Wz-LBYi448GoSFwaFgWFweGsoaoba0lH0jEA4EC0Hsznc3n8xHsYIi0FixmYyWfdoygk8olwKXtUmg+C4IzYaQaFaDOlohk2JlYhUovxXIUqobAYBgIWkBh8MBMDAMEpUKA8u2BgP8VCSLUR9FgADaNigPz4M1szIAtOSoIDyLhpoRU3xs3QoAgYJmbEYEFBiNWALqBoTlPyGrrwLgMSg02xp-inEmd2TMmge1GJmx8WMuTDYobGoYBQG4SevO1-OBUReg9hwSTr9US4eg5ddbySFLqY-PHEJ+0ILB+PdDA9HjdT+2SI5v0F5j88g4He8aCsKNoPiuW4nsympQSacA9n2AwAOR0Ig9aoWBXQXg+LiwGMkEIY+4zPq+nYPh+oEkaqMGobOkjzhg2G0UalE4t2vbUAMSYACwAAx8fgABMAmrPgACcAAcADsTbiLRpg4cpD6qeenZ1g2XJfmqDoav+7T6Awz4wFcDLAEmAnZpJTaKHxpjZlZNl2SJjnObZihucAnZ4WA5pwOZMAAIL9vetrToxzGGWAXEoWAzQAJQ1AAfGAAAi5kAHR8FALBJRxXr+f6uUKPO3SwFgXA4DAgZcA8GD+eQQXxs1MAAPJ8AgRHyiOnyoNIZkPCFYXxnpUULoV3osFcRicFwcj+YwQJ4Dy4b+iUVw3HcDzxjNVBzeMIAxUVxAKEYUTkOoPJSHgAQyKwGEBgNQ1cvGZ0DElqUZdluX5YlHFtZ13WEsS55nFKzLwAAHo8xHtHmVKODROLSLkX7tNViDcjYaMxe0V6yEsDy8RMgWSDYClqRxeQY9wOBVn8uT42ARUBeZ-lWETB5gLkcCaEIC2NdITAaCZRByAofMlGgmAgThhMNgepOI9SNYHJT8bqUunY5EsgJ1Ny4UomzlbBI4yy+IGCD1g0QsoOgvBCAw5oNkIkh9lg0jBGAIzsPLD7eDb1KW+MetYAbN40DFUNwND4xrlgqxo9mierHk2ax9DMVo3TRW-tIa1gBEURgAA4joACSYAABI5HQwAAAqi-7Pi8-zOFY4zthp3jHEE1zSsk8mNi+hgmvU+96Pw6C+dHEXJeNRX1cAEr+oCcDADokvt5oncMzjad5CzivEwyKaulQQoTzi2u4Z2Lgu7k9aAl+c+F2GPIYeXVdgEkKRgDbovDKmh-5QHSgAIR5IEYI8Ys4JyWCJFOicRIZyzjnaexscRd0PogvuCtB5nxVpSNWVNb400wfvbGCCsCoOZv3TmMgh7nzJskGsWlGyUw4nfI0ZxOjGmUHEH2ylEqeHEAcXAWV2YPGaOFN4kg0I5BgFwVCrIkRgFQrgZRWiVGslVo4NCXAzI6NUZ0MeiiTFqKVAyVCSiexaNMQTesjY0ICTcdZESABmbMniBLYVMIlLKgIoBJSAA)

Is this impossible with ElectroDB?   What am I missing here?

**Comment by tywalch** [comment]
Is the map you're hoping will default with that list?
